### DISCUSSION TITLE
LCD Beep Setting Not Saving

### DISCUSSION THREAD

**Discussion by embraceentropy** [Jyers/Marlin#561] [General Discussion]
THANK YOU for adding the option to disable the selection beeps. However, the printer doesn't retain this setting after powering off. I've tried several times (saving each time) with no luck. All other settings appear to be saving without issue. For now, I'm just turning it off first thing when I start the printer.

**Comment by AngleV** [comment]
https://github.com/Jyers/Marlin/issues/553
